api/src/opentrons/protocol_api/robot_context.py

```diff
@@ -0,0 +1,93 @@
+from typing import NamedTuple, Union, Dict, Optional
+
+from opentrons.types import Mount, DeckLocation
+from opentrons.legacy_commands import publisher
+from opentrons.hardware_control import SyncHardwareAPI, types as hw_types
+
+from ._types import OffDeckType
+from .core.common import ProtocolCore
+
+
+class HardwareManager(NamedTuple):
+    """Back. compat. wrapper for a removed class called `HardwareManager`.
+
+    This interface will not be present in PAPIv3.
+    """
+
+    hardware: SyncHardwareAPI
+
+
+class RobotContext(publisher.CommandPublisher):
+    """
+    A context for exposing finer robot control movement.
+
+    The RobotContext class provides the objects, attributes, and methods that allow
+    you to control robot motor axes individually.
+
+    They can command the robot to perform an action, like moving to an absolute position,
+    controlling the gripper claw or pipette motors.
```

Comment:
```suggestion
    Its methods can command the robot to perform an action, like moving to an absolute position,
    controlling the gripper jaw, or moving individual pipette motors.
```
the claaaaaw (our standard terminology is "jaw").